lexibank_abvdoceanic.py

```diff
@@ -93,6 +104,11 @@ def cmd_makecldf(self, args):
                 if entry.cognacy and entry.cognacy.lower() in ('s', 'x'):
                     continue  # pragma: no cover
 
+                # Skip entries which appear in etc/ignore.tsv
+                lid = slug(wl.language.name, lowercase=False)
+                if ignore_list and (lid, entry.name) in ignore_list:
```

Comment:
This also simplifies this statement then. Furthermore, you should specify the concept as well, since it is possible that an entry is bad for concept X and good for concept Y, so we should provide the conceept information (CID) as well here.